### ISSUE TITLE
where is the batch normalization in the inceptionV2 code?

### ISSUE THREAD

**Issue by jamesben6688** [tensorflow/models#3189]
I do not think there is batch normalization layer in the inceptionV2 model. Is there anything wrong with the code?

` net = slim.conv2d(net, depth(64), [1, 1], scope=end_point,
                        weights_initializer=trunc_normal(0.1))`

Also, I haven't seen anything about the batch normalization in any place of the InceptionV2 code. Is there any settings about this?

**Comment by aselle** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
